packages/code-connect/components/Accordion/AccordionToggle.figma.tsx

```diff
@@ -0,0 +1,39 @@
+import figma from '@figma/code-connect';
+import { AccordionItem, AccordionToggle, AccordionContent } from '@patternfly/react-core';
+
+/**
+ * PatternFly Accordion component integration for Figma Code Connect
+ */
+
+figma.connect(
+  AccordionToggle,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=1423-687&m=dev',
+  {
+    props: {
+      // string
+      expandText: figma.string('Expand Text'),
+
+      // enums
+      toggleTextExpanded: figma.enum('State', {
+        Default: figma.string('Toggle Text'),
+        Hover: figma.string('Toggle Text'),
+        Expanded: figma.string('Toggle Text Expanded')
+      }),
+
+      // enum
+      isExpanded: figma.enum('State', {
+        Expanded: true
+      }),
+
+      children: figma.children('*')
+    },
+    example: (props) => (
+      <AccordionItem isExpanded={props.isExpanded}>
+        <AccordionToggle>{props.toggleTextExpanded}</AccordionToggle>
```

Comment:
The AccordionToggle should have an `id="<your-id-here>" or something like that. `id` is a required prop.

Reply:
I'd also add a comment here recommending they implement an `onClick` event to toggle the expanded state of this Accordion.

Reply:
+1

Reply:
The AccordionToggle should have an id="<your-id-here>" or something like that. `id` is a required prop.

I'd also add a comment here recommending they implement an onClick event to toggle the expanded state of this Accordion.